tests/cases/standalone/common/promql/scalar.result

```diff
@@ -0,0 +1,237 @@
++---------------------+--------------------------------+
+
+-- SQLNESS SORT_RESULT 3 1
+TQL EVAL (0, 15, '5s') scalar(host{host="host1"} + scalar(host{host="host2"}));
+
++---------------------+-----------------------------------+
+| ts                  | scalar(lhs.val + rhs.scalar(val)) |
++---------------------+-----------------------------------+
+| 1970-01-01T00:00:00 | 3                                 |
+| 1970-01-01T00:00:05 | 7                                 |
+| 1970-01-01T00:00:10 | 11                                |
+| 1970-01-01T00:00:15 | 15                                |
++---------------------+-----------------------------------+
+
+-- SQLNESS SORT_RESULT 3 1
+TQL EVAL (0, 15, '5s') scalar(scalar(host{host="host2"}) + host{host="host1"});
+
++---------------------+-----------------------------------+
+| ts                  | scalar(lhs.scalar(val) + rhs.val) |
++---------------------+-----------------------------------+
+| 1970-01-01T00:00:00 | 3                                 |
+| 1970-01-01T00:00:05 | 7                                 |
+| 1970-01-01T00:00:10 | 11                                |
+| 1970-01-01T00:00:15 | 15                                |
++---------------------+-----------------------------------+
+
+-- SQLNESS SORT_RESULT 3 1
+TQL EVAL (0, 15, '5s') scalar(host + scalar(host{host="host2"}));
+
++---------------------+-----------------------------------+
+| ts                  | scalar(lhs.val + rhs.scalar(val)) |
++---------------------+-----------------------------------+
+| 1970-01-01T00:00:00 | 3                                 |
+| 1970-01-01T00:00:00 | 4                                 |
+| 1970-01-01T00:00:05 | 7                                 |
+| 1970-01-01T00:00:05 | 8                                 |
+| 1970-01-01T00:00:10 | 11                                |
+| 1970-01-01T00:00:10 | 12                                |
+| 1970-01-01T00:00:15 | 15                                |
+| 1970-01-01T00:00:15 | 16                                |
++---------------------+-----------------------------------+
+
+-- SQLNESS SORT_RESULT 3 1
+TQL EVAL (0, 15, '5s') scalar(scalar(host{host="host2"}) + host);
+
++---------------------+-----------------------------------+
+| ts                  | scalar(lhs.scalar(val) + rhs.val) |
++---------------------+-----------------------------------+
+| 1970-01-01T00:00:00 | 3                                 |
+| 1970-01-01T00:00:00 | 4                                 |
+| 1970-01-01T00:00:05 | 7                                 |
+| 1970-01-01T00:00:05 | 8                                 |
+| 1970-01-01T00:00:10 | 11                                |
+| 1970-01-01T00:00:10 | 12                                |
+| 1970-01-01T00:00:15 | 15                                |
+| 1970-01-01T00:00:15 | 16                                |
++---------------------+-----------------------------------+
+
+-- case have multiple time series, scalar return NaN
```

Comment:
The NaN behavior is different than Prometheus. It will produce an point with timestamp and `NaN`

Reply:
The behavior has been adjust in new version, it should have the same `scalar` behavior in Prometheus.